File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/spring/model/UsuarioCasaDTO.java
package org.tfg.inhometfgcarloshernandez.spring.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UsuarioCasaDTO {
    private String nombre;
    private String estado;
    private String colorEstado;
    private String colorUsuario;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/UsuarioCasaDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model

import com.example.inhometfgandroidcarloshernandez.common.Constantes

data class UsuarioCasaDTO(
    val nombre: String = Constantes.NADA,
    val estado: String = Constantes.NADA,
    val colorEstado: String = Constantes.NADA,
    val colorUsuario: String = Constantes.NADA,
)